packages/design-tokens/design-token-previews/borders.mdx

```diff
@@ -0,0 +1,22 @@
+import borderStyleTokens from "../src/tokens/border-style.tokens.json";
```

Comment:
Not specifically for this file. It is more related to the folder. We are already inside the `design-tokens` folder. Calling this folder `design-tokens-previews` it is redundant. 

Following what we have on [storybook](https://storybook.js.org/docs/react/writing-docs/mdx) for mdx use case, I would call it `docs`.